core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -107,6 +111,9 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
   windowIsNarrow: boolean = false;
   profilePicturePngDataUrl!: string;
   profilePictureWebpDataUrl!: string;
+  numberOfUnreadThreads: number = 0;
```

Comment:
Should we call this "unreadThreadsCount"?

Reply:
Done